tests/utils.py

```diff
@@ -53,6 +53,7 @@ def run(coro):
     os.path.dirname(__file__), "ssl_cert_with_chain.pem"
 )
 SERVER_KEYFILE = os.path.join(os.path.dirname(__file__), "ssl_key.pem")
+SERVER_COMBINED = os.path.join(os.path.dirname(__file__), "cert_key.pem")
```

Comment:
Minor nit: `SERVER_COMBINEDFILE` for the variable and `ssl_combined.pem` for the file